ProjectFiles/Assets/Scripts/Gameplay/ParkingSpace.cs

```diff
@@ -64,5 +63,16 @@ void OnTriggerExit(Collider other)
             {
             }
         }
+
+        public void SetOccupied(int playerID)
```

Comment:
We've used properties with a getter/setter, then wrapped them with an explicit setter. You can just have the property implement this behaviour, e.g.

```C#
private int occupiedBy;
public int OccupiedBy 
{
    get 
    { 
        return occupiedBy; 
    }
    set 
    {
        mat.color = new Color(0, 1, 0, 0.3f);
        occupiedBy = value;
    }
}
```